EternalModLoader/EternalModLoader.cs

```diff
@@ -249,19 +250,112 @@ public static void ReplaceChunks(MemoryStream memoryStream, ResourceInfo resourc
             {
                 foreach (var mod in resourceInfo.ModList)
                 {
-                    var chunk = GetChunk(mod.Name, resourceInfo);
+                    ResourceChunk chunk;
 
-                    if (chunk == null)
+                    if (mod.isBlangJson)
                     {
-                        resourceInfo.ModListNew.Add(mod);
-                        continue;
+                        var modName = mod.Name;
+                        var modFilePathParts = modName.Split('/');
+                        var name = modName.Remove(0, modFilePathParts[0].Length + 1);
+                        mod.Name = name.Substring(0, name.Length - 4) + "blang";
+                        chunk = GetChunk(mod.Name, resourceInfo);
+
+                        if (chunk == null)
+                            continue;
+                    }
+                    else
+                    {
+                        chunk = GetChunk(mod.Name, resourceInfo);
+                        
+                        if (chunk == null)
+                        {
+                            resourceInfo.ModListNew.Add(mod);
+                            continue;
+                        }
                     }
 
                     memoryStream.Seek(chunk.FileOffset, SeekOrigin.Begin);
                     long fileOffset = binaryReader.ReadInt64();
                     long size = binaryReader.ReadInt64();
                     long sizeDiff = mod.FileBytes.Length - size;
 
+                    // If the mod is a blang file json, modify the .blang file
+                    if (mod.isBlangJson)
+                    {
+                        BlangJson blangJson;
+                        try
+                        {
+                            blangJson = JsonConvert.DeserializeObject<BlangJson>(Encoding.UTF8.GetString(mod.FileBytes));
+                            if (blangJson == null || blangJson.strings.Count == 0)
```

Comment:
Always use braces, even for 1 line conditions. Please follow the C# coding convention: https://docs.microsoft.com/es-es/dotnet/csharp/programming-guide/inside-a-program/coding-conventions